File: src/main/java/com/xellitix/commons/encoding/base64/Base64Module.java
package com.xellitix.commons.encoding.base64;

import com.google.inject.AbstractModule;

/**
 * {@link Base64} module.
 *
 * @author Grayson Kuhns
 */
public class Base64Module extends AbstractModule {

  /**
   * Configures the module.
   */
  @Override
  public void configure() {
    bind(Base64.class).to(JavaUtilBase64.class);
  }
}

File: src/main/kotlin/com/xellitix/commons/encoding/base64/Base64Module.kt
package com.xellitix.commons.encoding.base64

import dev.misfitlabs.kotlinguice4.KotlinModule

/**
 * [Base64] Guice module.
 */
class Base64Module : KotlinModule() {

    override fun configure() {
        bind<Base64>()
    }
}
